specification/storage/resource-manager/Microsoft.Storage/storageDataManagement/preview/2024-11-01-preview/examples/Connectors_CreateOrUpdate.json

```diff
@@ -0,0 +1,84 @@
+{
+  "operationId": "Connectors_CreateOrUpdate",
```

Comment:
Does 2024-11-01-preview API version only has the connectors APIs?
If it support other storage APIs, like account, blob,.... related APIS, should first copy from the latest API version, then add the new API changes.